src/TestQuaternion.java

```diff
@@ -41,7 +27,31 @@ public void testEquals() {
         assertEquals(a, new Quaternion(1, 0, -1, 5));
         assertEquals(a, a);
         assertFalse(a.equals(new Quaternion(-1, 0, 0, 5)));
+    }
 
+    /**
+     * Tester for the constructors.
+     */
+    public void testConstructors() {
+        double tol = 1e-10; // Tolerance.
```

Comment:
Abbreviating names is generally not a good practice (In particular if you have to write a comment to explain it :)).